
### Allow use of RTL_USE_NEW_INTR_API that allows support for more
### modern MSI/MSIX IRQ kernel support

--- a/src/r8125.h	2022-03-31 07:27:49.000000000 +0000
+++ b/src/r8125.h	2022-05-06 02:36:55.000000000 +0000
@@ -51,11 +51,9 @@
 typedef int netdev_tx_t;
 #endif
 
-/*
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(4,12,0)&& !defined(ENABLE_LIB_SUPPORT)
 #define RTL_USE_NEW_INTR_API
 #endif
-*/
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,22)
 #define skb_transport_offset(skb) (skb->h.raw - skb->data)
